### PR TITLE
Update to support more than just 1204

### PR DESCRIPTION
**Pull request by vinnyfuria**
Transcripts:

```
spire@glas-sled01-pc01:~$ git clone git@github.com:nsat/uhd.git
Cloning into 'uhd'...
remote: Counting objects: 44175, done.
remote: Compressing objects: 100% (6/6), done.
remote: Total 44175 (delta 0), reused 1 (delta 0), pack-reused 44169
Receiving objects: 100% (44175/44175), 92.05 MiB | 6.95 MiB/s, done.
Resolving deltas: 100% (33145/33145), done.
spire@glas-sled01-pc01:~/uhd$ git checkout vinny_1604
Branch vinny_1604 set up to track remote branch vinny_1604 from origin.
Switched to a new branch 'vinny_1604'
```

14.04 and 16.04 Builds: https://gist.github.com/f3014c6a9ff8d10ed340db34dbf62f4f

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+BUILD INSTRUCTIONS
+==================
```
Comment (shainerz) [inline BUILD.md]: should we not just update `README.md`, rather than have build instructions in multiple places?

Comment (shainerz) [reply BUILD.md]: or reference this from README

File: BUILD.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+```
+
+# Building for Ubuntu 12.04:
+
+## Build Instructions
+```
+git clone git@github.com:nsat/uhd.git
+docker run -v ~/uhd:/uhd -it --entrypoint bash ubuntu:12.04
+apt-get update > /dev/null
+apt-get install -y git python-cheetah cmake libboost-all-dev lsb-core > /dev/null
```
Comment (shainerz) [inline BUILD.md]: is `libboost-all-dev` correct?

**Review by shainerz — COMMENTED** [review]
couple comments

File: BUILD.md (hunk 2 above)
Comment (vinnyfuria) [reply BUILD.md]: Yup, it's only one specific version of boost that is broken with gnuradio that happens to be the default in 12.04, hence the need to call out a specific version there.

**Review by shainerz — APPROVED** [review]
LGTM

**Pull request merged**